tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -713,3 +715,50 @@ fn test_read_index_after_write() {
             >= applied_index
     );
 }
+
+#[test]
+fn test_infinite_lease() {
```

Comment:
should add a test logic that normal write and read index would help skip the renew logic?
As this test only tests the case when there's no read or write traffic during max lease time,

Reply:
Added to `test_infinite_lease`. Thanks